### PR TITLE
Added self to contributors, fixed typo in share show page ('markdwon'…

### PR DESCRIPTION
**Pull request by shanekrusen**
… -> 'markdown'), added first three comments for share to shares index.

### CODE REVIEW

**Comment by kenmazaika** [comment]
Nice!  Looks good, but it seems to have a merge conflict.  If you can resolve the conficts (explained [here](http://thefirehoseproject.com/advanced-git-topics/5)) we can merge it.

**Comment by shanekrusen** [comment]
I think that should do it.

File: app/views/shares/index.html.erb [hunk 1]
```diff
@@ -25,6 +35,7 @@
         </div>
       </div>
     <% end %>
+>>>>>>> master
```
Comment (kenmazaika) [inline app/views/shares/index.html.erb]: I think this line of code needs to be deleted because it was added as part of a merge conflict.

Comment (shanekrusen) [inline app/views/shares/index.html.erb]: Oops. I must have just missed it. On it.

**Pull request merged**